app/app/explore/pools/[asset]/PoolDetail.tsx

```diff
@@ -85,26 +148,54 @@ export default function PoolDetail({ pool, runePriceUSD }: PoolDetailProps) {
                 YIELD
```

Comment:
Need to change this to Gain for now (see linear [comment](https://linear.app/project-chaos/issue/YLD-69/pool-details-add-liquidity#comment-3671c230))